pkg/printers/codeclimate.go

```diff
@@ -48,7 +54,7 @@ func (p CodeClimate) Print(issues []result.Issue) error {
 		codeClimateIssue.Fingerprint = issue.Fingerprint()
 		codeClimateIssue.Severity = defaultCodeClimateSeverity
 
-		if issue.Severity != "" {
+		if issue.Severity != "" && slices.Contains(p.allowedSeverities, issue.Severity) {
```

Comment:
It's up to you but this is enough


```suggestion
		if slices.Contains(p.allowedSeverities, issue.Severity) {
```


https://go.dev/play/p/7saiasJ_VB8